src/components/Routing.js

```diff
@@ -365,7 +366,34 @@ class Routing extends Component {
                             <Route
                                 exact
                                 path='/auto-import-seed-phrase'
-                                component={AutoImport}
+                                render={() => {
```

Comment:
`render()` receives `routeProps` so we should prefer using those over the `window` object :)
routeProps are: `{ match, location, history}`